owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -179,6 +179,10 @@ public class ReceiveExternalFilesActivity extends FileActivity
     private OCSharedPreferencesProvider sharedPreferencesProvider;
     private OCSpace personalSpace;
 
+    private boolean isInSpacesList;
+
+    private MenuItem menuItem;
```

Comment:
Do we really need this as an attribute of the class? We're just using it in one place 🤔

Reply:
In this case, `isInSpacesList` must to be an attribute because it's used in `onCreateOptionsMenu` which is part of lifecycle activity. On the other hand, I have changed `menuItem` because it's use only once in `onCreateOptionsMenu` function and now it's a local variable.

Reply:
Ok. I will keep that in my mind for next time. 👍